.github/workflows/CI.yml

```diff
@@ -13,14 +13,17 @@ jobs:
       fail-fast: false
       matrix:
         group:
+          #fixes 682
+          -"Bpinn ODE solver"
```

Comment:
this should be ODEBPINN with no quotes

Reply:
yeah my bad fixed it(ended up pushing it without changing)